projects/packages/backup/src/js/components/Backups.js

```diff
@@ -214,14 +214,26 @@ const Backups = () => {
 						</div>
 					) }
 					{ ! stats.warnings && (
-						<a
-							className="button is-full-width"
-							href={ getRedirectUrl( 'jetpack-backup', { site: domain } ) }
-							target="_blank"
-							rel="noreferrer"
-						>
-							{ __( 'See all your backups', 'jetpack-backup-pkg' ) }
-						</a>
+						<>
+							<a
+								className="button is-full-width"
+								href={ getRedirectUrl( 'jetpack-backup', { site: domain } ) }
+								target="_blank"
+								rel="noreferrer"
+							>
+								{ __( 'See all your backups', 'jetpack-backup-pkg' ) }
+							</a>
+							<small>
+								<a
+									className=""
+									href={ getRedirectUrl( 'backup-plugin-activity-log', { site: domain } ) }
+									target="_blank"
+									rel="noreferrer"
+								>
+									{ __( 'Or view your most recent restore point', 'jetpack-backup-pkg' ) }
+								</a>
+							</small>
```

Comment:
Just like above, we can build a single string here as well, and also use `ExternalLink`.